articles/css-in-js-solutions/index.md

```diff
@@ -0,0 +1,121 @@
+* Easy Modification: With JSS, making changes becomes more comfortable, because you only need to run down on where you declared the CSS rules and make necessary changes if any within the variable or function, and changes take effect where the variable or functions are applied. 
+
+Conclusively, JSS can be easily integrated into your React application. Follow this [guide](https://cssinjs.org/setup) to get started with JSS, and you can see a live code [here](https://codesandbox.io/s/z21lpmvv33) of CSS-in-JS in action with JSS.
+
+2. Styled-Components: Styled-components are one of the most used CSS-in-JS solutions in app development. It ensures the usage of ES6 and CSS to style your application with template literals and CSS; you can easily style app components.
+With an intense focus on improving user experience, styled-components are designed to enhance CSS styling for React app components. The following are the benefits of using styled-components in your application:
+* CSS Automation: By using styled-components as your preferred CSS-in-JS solution, it keeps track of which components are active and rendered on the page to inject all styles related to that component automatically, which increases the speed of loading your application.
+* Maintenance: As with other CSS-in-JS solutions, you can maintain your CSS codebase easily and track styles affecting your component without having to check through multiple CSS files.
+* CSS Deletion: Deleting CSS styles is made more comfortable, styled-components make it evident as to which component is unused, and deleting such components automatically deletes all styles related to that component.
+
+The above are some of the benefits of using styled-components as your preferred choice of CSS-in-JS solutions. You can view how styled-components handle CSS in JavaScript from [here](https://github.com/styled-components/styled-components#example), and you can also follow this [guide](https://www.styled-components.com/docs/basics#installation) to installing styled-components in your React application. 
+
+3. Styletron: Styletron is another CSS-in-JS library that works with plain JS, React, and other frontend frameworks. 
+Styletron has some great features which will be examined shortly, but the distinct feature that sets Styletron apart from other CSS-in-JS solutions is the ability to use media queries and other CSS selectors which couldn’t be done via inline styles.
+
+Well, apart from the cool feature of declaring media queries, here are some of the benefits of using Styletron:
+* Performance: To ensure high performance, Styletron follows the critical rendering concept by generating styles when needed by the demanding components. With this, browsers only load CSS rules that are currently in use rather than processing all CSS rules in a go.
```

Comment:
in **one** go. Please fix this, @CodeLeom since I've already mentioned it previously.